nautobot/extras/models/groups.py

```diff
@@ -43,12 +63,21 @@ class StaticGroup(PrimaryModel):
     content_type = models.ForeignKey(
         to=ContentType,
         on_delete=models.CASCADE,
-        related_name="static_groups",
+        related_name="static_groups_set",
```

Comment:
Same question here

Reply:
Intentional - `.static_groups` should always be the groups that an object is the member of. We don't support static groups of ContentType at this time but in case that's ever something we add, trying to preemptively avoid a name collision here.